sql_generators/glean_usage/templates/event_flow_monitoring_aggregates_v1.script.sql

```diff
@@ -39,12 +39,41 @@ CREATE TEMP TABLE
             UNNEST(event_extra) AS ext
           WHERE
             DATE(submission_timestamp) = @submission_date
-            {% if app['bq_dataset_family'] in ["accounts_frontend", "accounts_backend"] %}
-            AND metrics.string.session_flow_id IS NOT NULL
-            AND metrics.string.session_flow_id != ""
-            {% else %}
             AND ext.key = "flow_id"
-            {% endif %}
+        {% elif app['bq_dataset_family'] in ["accounts_frontend", "accounts_backend"] %}
+          (WITH events_unnested_with_metrics AS (
+            -- events_unnested views do not have metrics, accounts send flow_id in a string metric
+            -- so we need to unnest with metrics here
+            SELECT
+            e.* EXCEPT (events),
+            event.timestamp AS event_timestamp,
+            event.category AS event_category,
+            event.name AS event_name,
+            event.extra AS event_extra
+          FROM
+            `moz-fx-data-shared-prod.{{ app['app_name'] }}.events` e
+          CROSS JOIN
```

Comment:
Just confirming that rows without evetns are not required (LEFT JOIN).

Reply:
We want to use CROSS JOIN here, like in events_unnested view: https://github.com/mozilla/bigquery-etl/blob/79127d068140a17e9f7e86d160b8011a75ae48ad/sql_generators/glean_usage/templates/cross_channel_events_unnested.view.sql#L70-L71